requirements-dev.txt

```
@@ -1,5 +1,5 @@
 pip~=22.0
-git+https://github.com/PennyLaneAI/pennylane.git@master
+git+https://github.com/PennyLaneAI/pennylane.git@pauli-swap-order
```

```suggestion
git+https://github.com/PennyLaneAI/pennylane.git@master
```

Once the Pennylane PR is merged.